lib/consyncful/sync.rb

```diff
@@ -7,33 +7,52 @@
 require 'hooks'
 
 module Consyncful
+  ##
+  # A mongoid model that stores the state of a syncronisation feed. Stores the
+  # next URL provided by Contentfuls Sync API.
+  #
+  # Sync's are affectivly singletons,
+  # there should only ever be one in the database
+  #
+  # Is also the entrypoint of a Syncronization run
+  #
+
   class Sync
     include Mongoid::Document
     include Hooks
 
     define_hook :before_run
     define_hook :after_run
 
-    DEFAULT_LOCALE = 'en-NZ'
-
     field :next_url
     field :last_run_at, type: DateTime
 
     def self.latest
       last || new
     end
 
+    ##
+    # Delete the previous sync chains from database and create a fresh one.
+    # Used to completely resync all items from Contentful.
+    #
     def self.fresh
       destroy_all
       latest
     end
 
+    ##
+    # Makes sure that the database contains only records that have been provided
+    # during this chain of syncronisation.
+
     def drop_stale
       stale = Base.where(:sync_id.ne => id, :sync_id.exists => true)
       puts Rainbow("Dropping #{stale.count} records that haven't been touched in this sync").red
       stale.destroy
     end
 
+    ##
+    # Entry point to a syncronization run. Is reponsible for updating Sync state
+
```

Comment:
I probably wouldn't have a newline between a method comment and the definition. It decouples them. People sometimes write "section comments", and they usually have space around them (maybe that's not super common though?).